sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -74,6 +88,17 @@
 })
 public class BigQueryUtils {
 
+  public static Descriptor getDescriptorFromTableSchema(TableSchema jsonSchema)
```

Comment:
Probably we should add some comments to document the API of the public util functions here.

Reply:
done